### ISSUE TITLE
[0781 PS] A11Y issue - Hint text not programmatically associated with form input

### ISSUE THREAD

**Issue by lisacapaccioli** [department-of-veterans-affairs/va.gov-team#107759]
See: https://github.com/department-of-veterans-affairs/va.gov-team/issues/107325#issuecomment-2806871520

### Issue: Hint text not programmatically associated with form input

* **Page:** [Upload VA Form 21-0781](https://staging.va.gov/disability/file-disability-claim-form-21-526ez/mental-health-form-0781/upload)  
* **Device:** Mac  
* **Orientation:** landscape  
* **Expected severity level:** 2  
* **Expected experience standard:** User has enough information to complete a task  
* **Description:**   
  * The hint text isn’t programmatically associated with the form input (the Upload button): !
![A form control, Select a file to upload, with hint text and a button. The hint text is visually related to the button, but doesn't announce when the button has focus.](https://github.com/user-attachments/assets/29c57d3a-1164-4bed-b83e-6ca4ac18495f)
  * Screen reader users using TAB to access form inputs won’t hear “You can upload a .pdf, .jpg….” announced  
* **How to recreate:**   
  * Turn on screen reader  
  * TAB to the “Upload” button  
* **Recommendation:** Generally, the way to associate text like this is  
  * Add an id to the paragraph containing the hint text (say “\<p id=”hint”\>You can upload….\</p\>”)  
  * Reference that id in the form input using aria-describedby (\<button aria-describedby=”hint”\>)  
  * Since you’re using some custom code here, you’ll need to figure out how exactly to make that association work.

###  Tasks
- [ ] 

###  Acceptance
- [ ]
